### PR TITLE
feat(web): adding support for Wappalyzer in Recon page

### LINKED ISSUE #183: [3JH] Amelioration et Implémentation de nouvel API à la page recon
## Describe the issue
c'est une feature qui vise à améliorer la page recon avec un nouveau api

## Resource request
Aura besoin de communiquer avec le backend
Y'aura une nouvelle Figma de cette page, attendre l'issue #182 
Voici un example d'intégration api https://gist.github.com/9b/f5fe434bf9965d673963884b56d93d9a

## Definition of done
**UI** (1.5JH)
- [x] Change le tableau de recon à deux parties (recon+wappalyzer)
- [x] Respecter la style de Figma
- [x] Barre de recherche de wapperlyser ajouté sur la page
- [x] Bouton 'search' ajouter à côté de input barre
---- 
**API** (1.5JH)
- [x] Intégration d'api de [wapperlyzer](https://www.wappalyzer.com/api/) au front

## JEH
3

### PR DESCRIPTION
**Pull request by claraxuxu**
# Description

This PR is to update the recon page by adding a new api which names 'Wappalyzer'
https://stackoverflow.com/a/44438733 

# Fixes

Specify which issue this PR targets, like so:

- fixes #183 

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it.

# Checklist

- [x] I have assigned this PR to myself
- [ ] I have added @vrn-sh/members to the reviewers, or specific members of the team
- [x] I have added the needed labels
- [x] I have tested this code
- [x] I have added / updated tests (unit / functionals / end-to-end / ...)
- [x] I have updated the README and other relevant documents (guides...)

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it.

### CODE REVIEW

**Comment by claraxuxu** [comment]
In Progess

**Comment by djnnvx** [comment]
you may need to manually download the submodules in the github action, in order to fix the CI

File: web/src/assets/strings/en/wappa_test.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
+{
```
Comment (djnnvx) [inline web/src/assets/strings/en/wappa_test.json]: you can delete this

**Review by djnnvx — COMMENTED** [review]
looks good, let's see if it works

**Comment by djnnvx** [comment]
is this OK to merge ? @claraxuxu

**Pull request merged**